frontend/src/app/modules/shared/loader/loader.component.ts

```diff
@@ -17,22 +18,38 @@ import {fadeInOut} from '../animations';
 
         <!-- NOTE: We use ngClass below to make sure that content-wrapper is hidden in CSS
              before Angular animations get applied. This way we don't get any temporary glitches. -->
-        <!-- TODO: Clean up common logic from the template -->
         <div class='content-wrapper'
-             [ngClass]="(loading$ | async) && renderHiddenBeforeLoaded ? 'hidden' : ''"
-             [@fadeInOut]="(loading$ | async) && renderHiddenBeforeLoaded ? 'hidden' : ''"
-             *ngIf="(loading$ | async) === false || renderHiddenBeforeLoaded">
+             [ngClass]="hidden$ | async"
```

Comment:
Private properties are not compatible with Angular AoT 
- https://angular.io/guide/aot-compiler
- https://github.com/angular/angular/issues/11978
> Identifier 'renderContent$' refers to a private member of the componentAngular

Reply:
Rzeczywiście, niezła podpowiedź. Domyślnie kompilator dla lokalnego developmentu działa w trybie JiT, więc ten błąd nie wyskakuje. Dopiero użycie kompilatora AOT wywala błędem przy zoptymizowanym buildzie. Postaram się dodać jakieś checki do pull requestów w takim razie, żeby takie błędy się nie powtarzały. Swoją drogą dzięki za linki, coś tam się nauczyłem więcej ;)

Reply:
Naprawione :)